src/utils/thumbnailer_utils.h

```diff
@@ -45,10 +45,10 @@ typedef std::unique_ptr<WebPPicture, void (*)(WebPPicture*)>
 
 namespace libwebp {
 
-enum UtilsStatus {
-  kOk = 0,       // On success.
-  kMemoryError,  // In case of memory error.
-  kGenericError  // For other errors.
+enum [[nodiscard]] UtilsStatus{
```

Comment:
This does not seem to be taken into account if `GenerateAnimationEqualQuality()` can be called without using its return value, I am unsure why, maybe there is a compile flag to enable or something

Reply:
You are right this is not the main Status enum, my bad.